### DISCUSSION TITLE
List descriptions

### DISCUSSION THREAD

**Discussion by ronisaacson** [blocklistproject/Lists#224] [General]
Hi there and thanks for your efforts! Would it be possible to provide a description for each of the lists? Some of them are obvious from the titles, but others not so much. For example: the Redirect and Abuse lists, I couldn't even guess what those block. Maybe you could use the Comment column in README.md to provide a brief description. Thank you!

**Comment by gap579137** [comment]
Sounds like a good idea. Working on it now.

**Reply by gap579137** [reply]
Done

**Reply by ronisaacson** [reply]
Awesome, thank you!

**Reply by iam-py-test** [reply]
Question: Is @gap579137 different from @blocklistproject or are they just the same person with two accounts?
Thanks!

**Reply by blocklistproject** [reply]
@blocklistproject is the projects account and @gap579137 is my personal account. Effectively the same, I use my personal account to work on my personal projects and @blocklistproject to work solely on this project.

**Reply by iam-py-test** [reply]
@blocklistproject thanks for answering my question. It is just that I have seen both commiting & opening issues so was curious.
